<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Complex Application</string>
  </property>
  <widget class="QWidget" name="mCentralWidget">
   <layout class="QGridLayout" name="mGridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mMenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>683</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="mMenuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="mActionExit"/>
   </widget>
   <widget class="QMenu" name="mMenuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="mActionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="mActionShowSampleList"/>
   </widget>
   <addaction name="mMenuFile"/>
   <addaction name="menu_Tools"/>
   <addaction name="mMenuHelp"/>
  </widget>
  <widget class="QDockWidget" name="mSampleDock">
   <property name="windowTitle">
    <string>Available Samples</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="mSampleWidget">
    <layout class="QGridLayout" name="mPluginDockLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="mSampleButtonLoad">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="mSampleButtonUnload">
       <property name="text">
        <string>Unload</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QListWidget" name="mSampleList">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="mStatusBar"/>
  <action name="mActionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit Application</string>
   </property>
  </action>
  <action name="mActionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="mActionShowSampleList">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Sample List</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qrc/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
